java/ql/src/semmle/code/java/frameworks/google/GoogleHttpClientApi.qll

```diff
@@ -18,16 +11,17 @@ private class TypeLiteralToParseAsFlowConfiguration extends DataFlow5::Configura
 
   override predicate isSource(DataFlow::Node source) { source.asExpr() instanceof TypeLiteral }
 
-  override predicate isSink(DataFlow::Node sink) {
-    exists(MethodAccess ma |
-      ma.getAnArgument() = sink.asExpr() and
-      ma.getMethod() instanceof ParseAsMethod
-    )
-  }
+  override predicate isSink(DataFlow::Node sink) { sinkNode(sink, "google-parse-as") }
```

Comment:
This isn't really a query sink - I think we'll want to revert this change. Just because `GoogleHttpClientApi.qll` happens to have an internal use of the data-flow library that doesn't mean that we want to represent its sink through a csv model.